packages/multi-entry/types/index.d.ts

```diff
@@ -0,0 +1,17 @@
+import { Plugin } from "rollup";
+
+export interface Options {
+  /**
+   * If `true`, instructs the plugin to export named exports to the bundle from all
+   * entries. If `false`, the plugin will not export any entry exports to the bundle.
+   * This can be useful when wanting to combine code from multiple entry files, but
+   * not necessarily to export each entry file's exports.
+   * @default true
+   */
+  exports?: boolean;
```

Comment:
The options type can also be the `input` type: string, string array, or object with includes/excludes. See the [configure function](https://github.com/rollup/plugins/blob/master/packages/multi-entry/src/index.js#L12)

Reply:
Ah ok, that contradicts the readme, that should be updated too then

https://github.com/rollup/plugins/tree/master/packages/multi-entry#options

Reply:
@shellscape Is this intended behaviour? If so I agree we should update the README

Reply:
The README isn't necessarily wrong, but it needs an update to be more in line with other plugins. Specifically around `include` and `exclude`